bib/data/defs/fedora-40.yaml

```diff
@@ -75,7 +75,6 @@ anaconda-iso:
     - perl-interpreter
     - pigz
     - plymouth
-    - python3-pam
```

Comment:
Unsure about this removal, I see it mentioned in the Fedora IoT issue but I don't see `python3-pam` as a dep of `-img-deps`?

Reply:
Perhaps the removal of `python3-pam` should only be applied on Fedora 42?

Reply:
Yes, it is a dependency in the versions that require it  (Fedora 42 at this point):
```
sudo dnf repoquery --requires anaconda-install-img-deps
Updating and loading repositories:
 Fedora rawhide openh264 (From Cisco) - x86_64                                                                                                                                           100% |   7.7 KiB/s |   6.0 KiB |  00m01s
 Fedora - Rawhide - Developmental packages for the next Fedora release                                                                                                                   100% |  10.9 MiB/s |  21.9 MiB |  00m02s
Repositories loaded.
anaconda-install-env-deps = 42.13-1.fc42
brltty
createrepo_c
device-mapper-multipath
fcoe-utils >= 1.0.12-3.20100323git
gnome-kiosk
gnome-remote-desktop
gsettings-desktop-schemas
kdump-anaconda-addon
librsvg2
mesa-dri-drivers
nm-connection-editor
nvme-cli
openssl
ostree
podman
python3-pam
rpm-ostree >= 2023.2
skopeo
zram-generator-defaults
```